guides-markdown/multisig-cli.md

```diff
@@ -0,0 +1,463 @@
+# Multisig on the command line
+```post-author
+Matthew Zipkin
+```
+
+```post-description
+Throw a Multisig party for yourself and some friends using simple command line tools.
+```
+
+## Introduction
+
+The goal of this guide is demonstrate a multi-signature collaboration between bcoin
+users who may or may not trust each other. We will use only terminal commands to 
+illustrate the process. For a deeper dive into how multisig transactions are constructed
+in JavaScript with the library, see
+[Creating Multi-signature Transactions](https://bcoin.io/guides/multisig-tx.html).
+All the commands in this guide (and so many others!) are described in the
+[API docs](https://bcoin.io/api-docs). There are important details in those docs
+regarding API keys, wallet tokens, and other security measures that are essential
+for using bcoin on mainnet. For this guide, we will just play in regtest mode and
+skip over those steps.
+
```

Comment:
Maybe a dependencies section that says install bcoin and jq?

Reply:
I'll add a line explaining `jq`, but I don't think we need to start off every bcoin guide with "install bcoin"